inkscape/svg2shenzhen/export.py

```diff
@@ -568,7 +568,7 @@ def openKicad(self, kicad_file_path):
         else:
             command = "start %s" % (kicad_file_path)
 
-        return subprocess.Popen(command.encode("utf-8"), shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        return subprocess.Popen(command, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
There's an `inkex.commands` module which may (or may not) be helpful, so you don't have to use Popen directly.

Reply:
processExportLayer() method uses Popen to start multiple processes in parallel. I don't think the inkex.commands module supports this. Suggest we keep Popen to avoid introducing multiple methods for starting a process.